overlord/devicestate/firstboot.go

```diff
@@ -37,7 +38,11 @@ import (
 	"github.com/snapcore/snapd/timings"
 )
 
-var errNothingToDo = errors.New("nothing to do")
+var (
+	errNothingToDo = errors.New("nothing to do")
```

Comment:
nitpick^100: I would not mix in the same `var ()` block variables that are used just for mocking, with other variables. Or I'd add a comment to clearly identify those that are creating for being mocked.

Reply:
I agree on this